fdbclient/BlobWorkerInterface.h

```diff
@@ -51,7 +53,14 @@ struct BlobWorkerInterface {
 
 	template <class Archive>
 	void serialize(Archive& ar) {
-		serializer(ar, waitFailure, blobGranuleFileRequest, assignBlobRangeRequest, locality, myId);
+		serializer(ar,
+		           waitFailure,
+		           blobGranuleFileRequest,
+		           assignBlobRangeRequest,
+		           revokeBlobRangeRequest,
+		           granuleStatusStreamRequest,
```

Comment:
Somehow, we were able to send requests over these streams without serializing before... 😮 
Maybe because BM/BW were on the same process? and now they're spread out across diff processes?

Reply:
Yeah that's probably it. Local sending works differently than remote sending.